pydantic/types.py

```diff
@@ -872,3 +873,31 @@ def to(self, unit: str) -> float:
             raise errors.InvalidByteSizeUnit(unit=unit)
 
         return self / unit_div
+
+
+if TYPE_CHECKING:
+    StrictUnion = Union
+else:
+
+    class StrictUnionMeta(type):
```

Comment:
I guess this doesn't need to go inside the `if not TYPE_CHECKING`?

Reply:
I guess not